### DISCUSSION TITLE
Accuracy threshold on pass medals?

### DISCUSSION THREAD

**Discussion by LunaticaUWU** [ppy/osu#25384] [Ideas]
Hey. Personally, I think lazer by _"removing"_ notelock is making **star pass medals** many times easier. 
After release I suggest adding accuracy thresold to pass medals (example: Pass 10* map with accuracy over 65%).
Share your thoughts

**Comment by Rekunan** [comment]
We'd have to add a threshold for FC medals then, since sliderends no longer contribute to max combo.

**Reply by bdach** [reply]
> since sliderends no longer contribute to max combo

[outdated information](https://github.com/ppy/osu/pull/25342)

**Reply by Rekunan** [reply]
Ooo interesting. Is it like stable where if you drop a sliderend it's -1 combo, or does it always give sliderend combo?

**Reply by bdach** [reply]
It's not like stable _yet_. But at least max combo will be the same for classic and not-classic scores.

**Reply by LunaticaUWU** [reply]
Community always was telling scores were FC'd if no misses/sliderbreaks occured no matter of sliderends.

**Reply by Rekunan** [reply]
That's a good point yeah, I was just bringing it up since it's been a barrier for FC medals.
